lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,203 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
```

Comment:
Ahhh, I see now. Sorry, the function statement at the top made me think there was.